mqt/qusat/bindings.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+* This file is part of MQT QuSAT library which is released under the MIT license.
+* See file README.md or go to https://github.com/lucasberent/qsatencoder for more information.
+*/
+
+#include "SatEncoder.hpp"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+#include "pybind11_json/pybind11_json.hpp"
+#include "qiskit/QuantumCircuit.hpp"
+
+namespace py = pybind11;
+namespace nl = nlohmann;
+using namespace pybind11::literals;
+
+bool checkEquivalent(const py::object&              circ1,
```

Comment:
i think the interface of this function can be improved a bit.
just returning whether the circuits are equivalent is hardly enough.
you would typically expect to get all the statistics such as runtime, clauses, etc. as well.
this can be realized by exposing the results structure to Python.